lib/Basics/HybridLogicalClock.h

```diff
@@ -124,6 +123,7 @@ class HybridLogicalClock {
   static std::pair<size_t, size_t> encodeTimeStamp(uint64_t t, char* r) {
```

Comment:
Though this is possible, the change quickly gets out of hand, because it wouldn't be limited to just this function, but also `encodeTimeStampToValuePair(uint64_t, char*)` (which is a caller of this function) and `RevisionId::toString(char*)` (also a caller) and `RevisionId::toValuePair(char*)`, which is a caller of `RevisionId::toString(char*)`.